.moban.yaml

```diff
@@ -28,7 +28,7 @@ entry_points:
 
 dependencies:
   - appdirs~=1.4
-  - cached_property~=1.4.3
+  - cached-property~=1.4.3
   - cli_helpers~=1.0.2
```

Comment:
This also needs to be changed to hyphen right?

Reply:
Yup; fixed thx, and also added moban sync